src/renderer/entities/wallet/model/constansts.ts

```diff
@@ -0,0 +1,20 @@
+import { WalletType } from '@renderer/shared/core';
+import { IconNames } from '@renderer/shared/ui/Icon/data';
+
+export const WalletIconNames: Record<`${WalletType}`, IconNames> = {
```

Comment:
[TS template literal](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) does something?
```suggestion
export const WalletIconNames: Record<WalletType, IconNames> = {
```

Reply:
removed